segmentation/include/pcl/segmentation/supervoxel_clustering.h

```diff
@@ -225,6 +228,12 @@ namespace pcl
       void
       setNormalImportance (float val);
 
+      /** \brief Set whether or not to use the single camera transform 
```

Comment:
Could you add at least a short description of what it is? (As far as I remember, this is not explained in the paper.)